integrations/extensions/starter-kits/search-with-milvus/search-with-watsonx-data-milvus.md

```diff
@@ -0,0 +1,109 @@
+IBM API key is used as the password for the Milvus service on watsonx.data. There are two options to get the API key to access watsonx.data:
+* **(Option 1) Service ID access**: Assign a watsonx.data access role to a service ID and create a new API key or use an existing one from this service ID. You can manage service IDs from the [IBM Cloud IAM Service IDs page](https://cloud.ibm.com/iam/serviceids). 
+* **(Option 2) User access**: Assign a watsonx.data access role to a user and create a new API key or use an existing one from this user. You can manage user access and roles either from the [IBM Cloud IAM Users page](https://cloud.ibm.com/iam/users) or within the watsonx.data console by following the [Managing user access](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-manage_access) doc.
+
+
+### Get other connection details
+Here are the steps to collect other Milvus connection details from the watsonx.data console:
+
+1. Go to the `Infrastructure manager` page.
+2. Click the Milvus service to open the `Details` page.
+3. Click on `View connect details` to view more connection details.
+4. Collect the service GRPC `host`, `port` and the SSL certificate.
+
+## Step 2: Ingest data into Milvus
+You can ingest data into Milvus vector database either through watsonx.ai or by using custom code.
+### Option 1: Ingest data through watsonx.ai
+#### Create a Milvus connection
+On the watsonx.ai Project Assets page, click on `New asset` --> choose `Connect a data source` --> choose `Milvus` --> click `Next` --> fill in the connection details and credentials as below --> `Test connection` --> click `Create`.
+
+<img src="./assets/create-milvus-connection.png" width="1080" height="574" />
+
+**NOTE: SSL certificate is optional. If you include a certificate, it has to be signed by a known certificate authority.**
+
+#### Create a vector index and upload documents
+On the watsonx.ai Project Assets page, using the Milvus connection created in the previous step, you can create a vector index and upload documents into it. Here are the steps:
+1. On the watsonx.ai Project Assets page, click on `New asset` --> choose `Ground gen AI with vectorized documents`.
+2. On the left-side panel, select `watsonx.data Milvus` as the vector store --> fill in name and description --> select the Milvus connection created earlier. 
+3. Select the Milvus connection created earlier -> Select `Database` and `Embeddings model` from the dropdowns --> click `Next`. For example,  
+<img src="./assets/create-milvus-index-watsonx-ai.png" width="1080" height="574" />
+
+4. Click `New collection` to create a new collection.
+5. Provide a unique collection name and choose files to add to the Milvus collection --> click `Create`.
+6. Once the document uploading process is done, you can start testing it in the prompt lab.
+
+See the following screen recording for more details: [./assets/create-milvus-index-watsonx-ai.mov](./assets/create-milvus-index-watsonx-ai.mov).
```

Comment:
Suggest to be more specific about why they would want to watch the screen recording.  Perhaps say: "The following screen recording is a demonstration of steps 1-6."